contracts/payment/LemonadeEscrowV1.sol

```diff
@@ -15,9 +15,9 @@ contract LemonadeEscrowV1 is
     PaymentSplitter,
     AccessControlEnumerable
 {
-    bool internal _closed;
-    uint16 internal _hostRefundPercent;
-    RefundPolicy[] internal _refundPolicies;
+    bool public _closed;
```

Comment:
If the variables are public then the names should not be prefixed with an underscore

Reply:
FYI I see in other conventions, they name their function params with underscore prefix and state variable as normal ones.